packages/ui5-config/src/index.ts

```diff
@@ -13,3 +13,5 @@ export {
     UI5ProxyConfig,
     UI5ProxyConfigTarget
 } from './types';
+
+export { errors as yamlErrors } from '@sap-ux/yaml';
```

Comment:
This is another place that could use some discussion. Here I re-export YAML errors so that consumers of ui5-config don't need to declare a dependency to `@sap-ux/yaml` just to use the error codes (like `odata-service-writer`)

Reply:
The question is still valid, even though the original comment is shown as outdated

Reply:
I think it makes sense to do that. I am currently looking at the `abap-deploy` task that uses the `axios-extension`. In my code I needed the type for `AxiosError` from `axios` and since it is part of the signature of one of the `axios-extension` functions, it made sense to me to also re-export it.